languages/python/oso/oso/exceptions.py

```diff
@@ -4,9 +4,9 @@ class AuthorizationError(Exception):
 
 class NotFoundError(AuthorizationError):
     """
-    Thrown by the ``authorize`` method of an ``Enforcer``. This error indicates
-    that the actor is not only not allowed to perform the given action, but also
-    is not allowed to ``"read"`` the given resource.
+    Thrown by the ``authorize`` method of an ``Oso`` instance. This error
+    indicates that the actor is not only not allowed to perform the given
+    action, but also is not allowed to ``"read"`` the given resource.
```

Comment:
nittiest nit, to account for cases where someone's specified a 'read' action that isn't `"read"`:

```suggestion
    action, but also is not allowed to read the given resource.
```

Reply:
Might keep this as-is. I kind of like defaulting to `"read"` in the docs, etc. Ideally, the custom read action is something most people shouldn't have to think about doing.

It is sort of analogous to how we've been documenting the errors: I typically write that `authorize` raises `NotFoundError` or `ForbiddenError` -- it's almost too complicated to explain that it raises "the not found error configured for your Oso instance".

Reply:
Cool, that makes sense